### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Mihndim2020**
This project is a clone of the New times article website  (https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0). The objective of the project is the implement(put into use, showcase, cement) the skills learnt in the technical curriculum of microverse, which has to do with the positioning of elements using the various positioning techniques. The positing techniques used for this project are: floats, and flex-box, relative, absolute, fixed.
Additional description about the project and its features.

### CODE REVIEW

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @Mihndim2020 and @promise-J, I am Abdulaziz Redi Ali, your TSE for this PR.  👏👏  I appreciate your hard work in replicating nytimes.com.   Kindly fix the following issues.

[REQUIRED]

- [ ] Fix web hint linting errors
- [ ] Add buttons to the right-most part of the navigation bar
- [ ] Align all elements in the page to the center

 
| Your Copy | Original | 
| :---:         |     :---:      |
|alignment_copy  | alignment_original |
|alignment2_copy |alignment2_original |

- [ ] The page shouldn't scroll horizontally fix the overflow and horizontal scrolling.
horizontal_scrolling
- [ ] Fix the alignment and styling of Editors Pick's section to look more like the original page

| Your Copy | Original | 
| :---:         |     :---:      |
|editors_pick_copy |editors_pick_original |

- [ ] The articles in the "Most popular"  and the "suggested articles" section in the bottom part of the page should be links

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @Mihndim2020 and @promise-J, I am Abdulaziz Redi Ali, your TSE for this PR.  👏👏  I appreciate your hard work in replicating nytimes.com.   Kindly fix the following issues.

[REQUIRED]

- [ ] Fix web hint linting errors
- [ ] Add buttons to the right-most part of the navigation bar
- [ ] Align all elements in the page to the center

 
| Your Copy | Original | 
| :---:         |     :---:      |
|alignment_copy  | alignment_original |
|alignment2_copy |alignment2_original |

- [ ] The page shouldn't scroll horizontally fix the overflow and horizontal scrolling.
horizontal_scrolling
- [ ] Fix the alignment and styling of Editors Pick's section to look more like the original page

| Your Copy | Original | 
| :---:         |     :---:      |
|editors_pick_copy |editors_pick_original |

- [ ] The articles in the "Most popular"  and the "suggested articles" section in the bottom part of the page should be links

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @Mihndim2020 :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this project can be approved.

## REQUIRED

* [ ] Please, make all the changes as requested by the previous TSE.
* [ ] Please, work on this:
**Yours**
image
**Original**
image
- [ ] Kindly make sure your page looks exactly like the [original](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)

### Make the requested changes and submit for another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,621 @@
...
+            </svg>
+            <div class="button">
+                <button type="button">SUBSCRIBE NOW</button>
+                <button type="button">LOGIN</button>
+            </div>
+        </nav>
+    </header>
+    <main class="container">
+            <h1 class="titie">Space Ripples Reveal Big Bang’s <br> Smoking Gun</h1>
+        <!-- the social section of the pages -->
```
Comment (ioilmio) [inline index.html]: Remove comments

File: index.html [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+        <aside class="socials">
+            <!-- a child content that makes styling the inner contents much easier -->
```
Comment (ioilmio) [inline index.html]: Remove comments

File: index.html [hunk 3]
```diff
@@ -0,0 +1,621 @@
...
+        <figure class="space-image">
+            <img src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-jumbo.jpg?quality=90&auto=webp"
+                alt="space image">
+            <figcaption>Alan Guth was one of the first figcaptionhysicists to hypothesize the existence of inflation,
+                which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang
+                13.8 billion years ago.Credit...Rick Friedman
+                for The New York Times</figcaption>
+        </figure>
+
+        <!-- the section about CAMBRIDGE with a date stamp...the long text -->
```
Comment (ioilmio) [inline index.html]: Remove comments

**Review by ioilmio — CHANGES_REQUESTED** [review]
Hello @Mihndim2020 @promise-J  👋🏽 
Nice job till now let's improve it 🦾 
Required changes 🔴 :

### Header section

- #### What happened to the bookmark icon? Please fix it

### Caption 

- #### Please align the caption correctly, for the big image;

### Theory of inflation 

- #### You are almost there, but some text is still overlapped, please fix it;

### Video
-  #### Should be centered along with its caption;

### Last image
- #### Caption and text form the paragraph are mixed please fix them;

### Footer 
- Editor's pick section is not as the original, please fix it;


# General
## Add node_modules folder to your .gitignore file, so it won't be pushed ot github;
## Fix your readme layout;
## Text in the page should be in the center, not jutified to center;

# For further info contact me on slack @illuminatosalvatore 

Happy coding 💻

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Jp Johnson Alisigwe** 
Good job with the improvements for this project :muscle:  :muscle:
Your page looks like the original :clap: :clap: :clap:  
There are just a few details to correct before you can go ahead:

- [ ] Add  screenshot in your README file
- [ ] Add a link to the Live Demo of your page
- [ ] The website logo, in the top navbar, should be a link
- [ ] "SPACE AND COSMOS" text, in the top navbar, should be a link
- [ ] In your CSS file, group selectors with the same style definition
- [ ] Your project should align elements with a combination of all three techniques (float, flex and grid). 
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Comment by promise-J** [comment]
Hello Oscar, we have done the changes as requested in your review above.
please could you get back to us on the project? thank you
https://github.com/Mihndim2020/Positioning-and-floating-elements/pull/1

**Review by Srjnnnn — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 
Hello team. Your page looks good but there are some things that need to be solved before moving on. :smile: 



- [ ]  Please align those social icons and give a bit of padding to them.



- [ ]  Please align the sentence as on the original page.



- [ ]  Please place the editor's section a bit more to left.



- [ ]  Please align this image on the center. 



- [ ]  There is an overlapping issue as you can see from the screenshot above. Kindly fix this issue.
- [ ] On your readme, put the license as a link, don't copy and paste all the license.

### Optional

- [ ]  You can work on the footer, decrease the size of it, and make it as on the original page.

Again, a good start and don't be discouraged from the reviews. It is to help you improve your skills. :+1: 

Please fix those mentioned issues and submit for a re-review after that.
If you have any questions about the review, feel free to reach out to me via Slack. :100: 
_Eyüp Sercan UYGUR_

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @Mihndim2020 and @promise-J , I am Abdulaziz Redi Ali, your TSE for this PR.  👏👏  I appreciate your hard work in replicating nytimes.com.   Kindly fix the following issues.

[REQUIRED]

1. - [ ] The editor's pick section should be before the "The Theory of Inflation".
2. - [ ] Implement the footer action button for the main article
3. - [ ] Fix the footer, it should have a similar font style to the original.

 
| # | Your Copy | Original | 
| :---:| :---:         |     :---:      |
| 1 | IMAGE 2021-01-16 13:21:07|Screenshot 2021-01-16 235622| 
| 2 | Screen Shot 2021-01-16 at 1 22 58 PM | Screen Shot 2021-01-16 at 1 01 12 PM |
| 3 | Screen Shot 2021-01-16 at 1 23 34 PM|Screen Shot 2021-01-16 at 1 24 01 PM |

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Comment by mimipeshy** [comment]
### STATUS : CHANGES REQUIRED :red_circle: :recycle: 

Hello @Mihndim2020 This is good to make it this far, your project is really nice, congratulations on the good work you have done so far. I liked that you have implemented the requested changes from the previous review, keep it up, and your project is really almost done. One change is required:

- [ ] Your README file format is distorted. You have entered the same information twice. Kindly check on it.
- [ ] The getting started section should contain instructions on how to set up your project locally. Kindly add them.

That's all :smile: , kindly submit another review when done.

_Feel free to reach out to me on slack if you have any questions or something is unclear, I will be glad to help_
Regards
Peris Ndanu

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello `Jp Johnson Alisigwe` :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] You need to add a description to this Pull Request
- [ ] Files not related to the project should not be committed and added to .gitignore (e.g. `node_modules`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [ ] You need to fix your readme file, the `author` and `license` sections are duplicated (present twice), fix them
- [ ] The bottom part of your clone, before the footer, the text is overlapping pictures, fix that.
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by promise-J** [comment]
[image: g1.PNG][image: g2.PNG][image: f1.PNG][image: f2.PNG]

**Comment by promise-J** [comment]
this is my view on the browser. both on chrome and on Mozilla fire fox. the
are both responsive.

**Comment by promise-J** [comment]
g1
g2
f1


these are the views on Mozilla and chrome for the specified sections.

**Comment by promise-J** [comment]
Fixed the redundancy in the README file.

**Comment by promise-J** [comment]
I added all files not related to the project to .gitignore (e.g. node_modules)
I fixed the readme file, the author and license sections that appeared twice was erased.
 Fixed the overlapping text in the footer.

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @Mihndim2020 and @promise-J  ! :wave:

#### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

 **Optional**
- [ ] Use [this](https://chrome.google.com/webstore/detail/font-picker/mmjbimgpcbaegjiieojddickpjbdkeej) chrome extension to help with the varying font-sizes and family required.

#### You may merge, good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**